api/urls.py

```diff
@@ -49,11 +49,11 @@
     url(r'^taxlot-columns/$', get_taxlot_columns, name='taxlot-columns'),
     url(r'^properties/(?P<property_pk>\d+)/cycles/(?P<cycle_pk>\d+)/$',
```

Comment:
This can be rewritten like this:
property_detail = Property.asView({
    'put': 'put',
    'get': 'get_property',
})
url(r'^properties/(?P<property_pk>\d+)/cycles/(?P<cycle_pk>\d+)/$', property_view, name='property')

which might be a little cleaner  if you don't mind shared names.